lib/Dialect/TritonGPU/Transforms/Utility.cpp

```diff
@@ -818,6 +819,24 @@ bool isPureUnaryInlineAsm(Operation *op) {
          inlineAsmOp.getPure();
 }
 
+int getComputeCapability(Operation *module) {
+  assert(module->hasAttr(triton::AttrTargetName) &&
+         "Expected a target attribute on the module operation");
+
+  StringAttr targetAttr =
+      cast<StringAttr>(module->getAttr(triton::AttrTargetName));
+
+  auto ref = targetAttr.strref();
```

Comment:
Suggest to use concrete type instead of auto here and at line 833

Reply:
done